ibis/omniscidb/tests/test_client.py

```diff
@@ -287,3 +287,97 @@ def test_cpu_execution_type(
 
     for mocked_method in mocked_methods:
         mocked_method.stop()
+
+
+@pytest.mark.parametrize(
+    'table_src',
+    [
+        ibis.schema(
+            [('a', 'polygon'), ('b', 'point'), ('c', 'int8'), ('d', 'double')]
+        ),
+        ibis.table(
+            schema=ibis.schema(
+                [
+                    ('index', 'int64'),
+                    ('Unnamed__0', 'int64'),
+                    ('id', 'int32'),
+                    ('bool_col', 'bool'),
+                    ('tinyint_col', 'int16'),
+                    ('smallint_col', 'int16'),
+                    ('int_col', 'int32'),
+                    ('bigint_col', 'int64'),
+                    ('float_col', 'float32'),
+                    ('double_col', 'double'),
+                    ('date_string_col', 'string'),
+                    ('string_col', 'string'),
+                    ('timestamp_col', 'timestamp'),
+                    ('year_', 'int32'),
+                    ('month_', 'int32'),
+                ]
+            ),
+            name='functional_alltypes',
+        ),
+        param(
+            pd.DataFrame(),
+            marks=pytest.mark.xfail(reason="Not implemented yet."),
+        ),
+        param(
+            pd.DataFrame([(1, 2.0, 'string'), (3, 4.0, 'string')]),
+            marks=pytest.mark.xfail(reason="Not implemented yet."),
+        ),
+    ],
+)
+def test_create_table(con, table_name, table_src):
+
+    if isinstance(table_src, ibis.Schema):
+        con.create_table(table_name, schema=table_src)
+    else:
+        table_src.name = table_name
+        con.create_table(table_name, obj=table_src)
+
+    assert con.exists_table(table_name)
+    if isinstance(table_src, ibis.Schema):
+        assert con.get_schema(table_name) == table_src
+    elif isinstance(table_src, pd.DataFrame):
+        assert con.get_schema(table_name) == ibis.schema(
+            types=table_src.dtypes()
+        )
+    elif isinstance(table_src, ibis.expr.types.TableExpr):
+        assert con.get_schema(table_name) == table_src.schema()
+
+
+@pytest.mark.parametrize('is_super', [False, True])
+def test_create_user(con, user, is_super):
+    name, password = user
+    con.create_user(name, password, is_super)
+
+    # if is_super=False, then it should throw an exception
+    try:
+        ibis.omniscidb.connect(
+            protocol=con.protocol,
+            host=con.host,
+            port=con.port,
+            user=name,
+            password=password,
+        )
+    except Exception:
```

Comment:
huh? what exactly is this testing? why do you have a try/except around it in a test? if you really need to test something like this, pls use a mock